server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/QuotaMailingTest.java

```diff
@@ -144,13 +157,15 @@ public void configurationShouldBeWellLoaded() throws Exception {
             .statusCode(200)
             .log().ifValidationFails()
             .body(ARGUMENTS + ".list.textBody",
-                hasItem(containsString("You currently occupy more than 10 % of the total size allocated to you")));
+                hasItem(containsString("You currently occupy more than 10 % of the total size allocated to you")))
```

Comment:
you can use hasItems when you have several conditions

Reply:
Well `hasItems` complained when used in combinaison with `containsString` matcher...

Reply:
It fails with that error `java.lang.IllegalArgumentException: You must supply at least one key and one value.`

Reply:
I had it working yesterday but don't bother with that.